synthetic_data/synthetic_data.py

```diff
@@ -269,8 +274,9 @@ def make_tabular_data(
     #        x_final = x_cont
 
     # rescale to -1 to 1
-    scaler = MinMaxScaler(feature_range=[-1, 1])
-    x_final = scaler.fit_transform(x_final)
+    if feature_range is not None:
```

Comment:
Previously all data was rescaled. Now all data is only rescaled if the feature_range parameter is set. Do we always want every feature to be rescaled with the same range? Not an issue with this implementation but perhaps a ticket worth discussing.